lib/registries/addon/drafts/draft/-components/register/component.ts

```diff
@@ -29,7 +30,7 @@ export default class Register extends Component.extend({
 
     // Private
     registration!: Registration;
-    rootNode?: Node;
+    rootNode!: NodeModel;
```

Comment:
`rootNode` is only set if the request to fetch `draftRegistration.branchedFrom` succeeds -- seems like we should keep the `?`. Good catch on the global [`Node`](https://developer.mozilla.org/en-US/docs/Web/API/Node), though 👍